packages/react-components/src/components/EndCallButton.tsx

```diff
@@ -1,12 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import React from 'react';
+import React, { useCallback } from 'react';
 import { DefaultButton, IButtonProps, Label, concatStyleSets, mergeStyles, useTheme } from '@fluentui/react';
 import { CallEndIcon } from '@fluentui/react-northstar';
 import { controlButtonLabelStyles, endCallControlButtonStyles } from './styles/ControlBar.styles';
 import { lightTheme, darkTheme } from '../theming/themes';
 import { isDarkThemed } from '../theming/themeUtils';
+import { useLocale } from '../localization';
+
+/**
+ * Strings of EndCallButton that can be overridden
+ */
+export interface EndCallButtonStrings {
+  /**
+   * Label of button
+   */
+  text: string;
```

Comment:
nit: I think this should be `label` or `labelText` for the buttons. So users can make the connection that this text is related to the showLabel prop